site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -0,0 +1,133 @@
+
+* <<Features>>
+* <<Common Applications>>
+* <<Entity Types Homepage>>
+* <<Entity Homepage>>
+
+=== Features
+
+==== Scalability
+- FusionAuth Entity Management supports large volumes of Entities right out of the box.
+- This is especially helpful for Internet of Things (IOT) devices; FusionAuth scales right alongside them.
+
+==== Typecasting
+- Entities can have types.
+- For example, `locks`, `cars`, `companies`, `corporate divisions`, `computers`, `APIs`
+- Entity Types can help define permissions.
+- You are limited only by your business need and imagination!
+
+==== Permissions aware
+- Permissions can be assigned to each Entity Type.
+- Entities can share permissions with other Entities (In OAuth parlance, this synonymous with the client credentials grant).
+- Users can have permissions to Entities.
+
+=== Common Applications
+- Corporate relationship modeling
+- Per use device permissions
+- Internet IoT
+
+====  Can't I just use a group?
+
+In some cases, Groups work as a model for such ideas as `customers`. However, the flexibility of Groups is limited by their lack of typecasting (very much needed as use cases evolve). Additionally, Groups do not have a hierarchical model or permissions functionality built in.
+
+//Here's a brief video covering some aspects of Entity Management:
+// Placeholder for a video in the future
+//
+//video::DaZbwrA7M90[youtube,width=560,height=315]
+
+=== Entity Types Homepage
+
+image::entity-types-homepage.png[Home Screen Entity Types,width=1200]
+
+Welcome to the Entity Types homepage! +
+
+Here you can: +
+
+[width="50%", cols="0, 3"]
+|===
+| icon:plus[role=ui-button green,type=fas] | *Create* a new Entity Type
+| icon:edit[role=ui-button blue,type=fas] | *Edit* a previously created Entity Type
+| icon:key-skeleton[role=ui-button black,type=fas] | *Manage Permissions* on Entity Type
+| icon:search[role=ui-button green,type=fas] | *View* the previously created Entity Type
+| icon:trash[role=ui-button,type=fas] | *Remove* the previously created Entity Type
+|===
+
+==== Add Entity Type form fields
+
+image::add-entity-type.png[Create an Entity Type,width=1200]
+
+Creating a new Entity Type is easy!
+
+Just complete the following fields:
+
+[.api]
+[field]#Id# [optional]#Optional#::
+An optional UUID. When this value is omitted, a unique Id will be generated automatically.
+
+[field]#Name# [required]#Required#::
+The name of the Entity Type. This value is for display purposes only and can be changed at any time.
+
+[field]#JSON Web Token Settings# [optional]#Optional#::
+If you need custom JWT signing algorithms. If left unchecked, this defaults to the tenant settings for that Entity Type.
+
+[field]#Permisions# [optional]#Optional#::
+Here you can add custom permissions. +
+Subfields include: `Name`, `Default: Y/N` and `Description`.
+
+=== Entity Homepage
+
+image::entity-homepage.png[Entity Homepage,width=1200]
+
+Welcome to the Entity homepage! +
```

Comment:
Maybe a bit too informal. We are more casual with blog posts, but for docs we have a tone of 'just the facts' :)